dcc-common-es/README.md

```diff
@@ -0,0 +1,12 @@
+ICGC DCC - Common Elasticsearch
+===
```

Comment:
You first :) 
https://raw.githubusercontent.com/icgc-dcc/dcc-common/develop/dcc-common-ega/README.md